.github/ISSUE_TEMPLATE/code_quality.md

```diff
@@ -2,7 +2,6 @@
 name: Code Quality Improvement Suggestion
 about: Suggest a change that does not add a feature
 labels: "🚦 status: awaiting triage"
-title: "<Replace this with the actual title>"
```

Comment:
Please don't remove the title, that is a required field.

Reply:
so sir do you mean it should show empty content , like:
title:
?

Reply:
No, I meant to leave the line unchanged. The way it was before.